guides/creating-a-custom-stepper-using-the-cdk-stepper.md

```diff
@@ -27,6 +27,9 @@ export class CustomStepperComponent extends CdkStepper {
 
   /** The index of the selected step. */
   selectedIndex: number;
+  
+  /** The step that is selected. */
+  selected: CdkStep;
```

Comment:
Doesn't this get inherited from `CdkStepper` already? Same goes for `selectedIndex` and `linear`.